x-pack/plugins/stack_connectors/server/connector_types/bedrock/bedrock.ts

```diff
@@ -141,7 +143,8 @@ export class BedrockConnector extends SubActionConnector<Config, Secrets> {
     const req = {
       // end prompt in "Assistant:" to avoid the model starting its message with "Assistant:"
       prompt: `${combinedMessages} \n\nAssistant:`,
-      max_tokens_to_sample: 300,
+      max_tokens_to_sample: 8191,
```

Comment:
does this number come from somewhere? seems random :)

Reply:
I can't find it in the docs, but when you hit an API with a number greater than 8191:
```
{
    "message": "Malformed input request: 8192 is not less or equal to 8191, please reformat your input and try again."
}
```